core/src/main/scala/com/rackspace/com/papi/components/checker/step/results/MultiFailResult.scala

```diff
@@ -87,5 +88,5 @@ class MultiFailResult(val fails : Array[Result], stepId : String) extends ErrorR
     }
   }
 
-  override def path : String = startPath+stepIDs.reduceLeft(_+" "+_)+" " + fails.map( _.path ).reduceLeft( _ + " " + _ ) + endPath+"*"+selectId(this)+"*"
+  override def path : String = startPath+super.stepIDs.reduceLeft(_+" "+_)+" " + fails.map( _.path ).reduceLeft( _ + " " + _ ) + endPath+"*"+selectId(this)+"*"
```

Comment:
In line 43 we add the super's stepIDs to the highest priority (i.e. head) stepIDs. If this combined version is used in the `path`, then the highest priority failure path will be listed twice. This is because we need it for other items, but the `path` shows all failure paths not just the highest priority.